File: Arrays/Easy/268/268.c
int missingNumber(int* nums, int numsSize){
    int sum = 0;
    int summation = numsSize * (numsSize + 1) / 2;
    for(int i = 0; i < numsSize; i++){
        sum += nums[i];
    }
    return summation - sum;
}

File: Arrays/Easy/268/268.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let n = nums.length
    let sum = 0
    for(let i = 0; i < n; i++){
        sum += nums[i]
    }
    n = n * (n + 1) / 2;
    return n - sum
};